generators/new-wp-project/templates/composer.json

```diff
@@ -16,7 +16,8 @@
     "wpackagist-plugin/broken-link-checker": "*",
     "wpackagist-plugin/postmark-approved-wordpress-plugin": "*",
     "wpackagist-plugin/mainwp-child": "*",
-    "wpackagist-plugin/akismet": "*"
+    "wpackagist-plugin/akismet": "*",
+    "wpackagist-plugin/rest-api": "dev-trunk"
```

Comment:
I think dev-trunk is the stable version... at least I don't see any other viable options: http://snaps.getmoxied.net/412t0F3y3Y01